src/test/java/com/google/api/codegen/testdata/csharp_gapic_client_library.baseline

```diff
@@ -1748,17 +1774,22 @@ namespace Google.Example.Library.V1
         public virtual Task<PublishSeriesResponse> PublishSeriesAsync(
             Shelf shelf,
             IEnumerable<Book> books,
-            uint edition,
+            uint? edition,
```

Comment:
Providing a null value will always logically be the same as setting it to 0 for value types. Arguably clearer though.